src/skmatter/metrics/_reconstruction_measures.py

```diff
@@ -496,19 +444,23 @@ def local_reconstruction_error(
         Number of neighbour points used to compute the local reconstruction weight for
         each sample/point.
     train_idx : numpy.ndarray, dtype=int, default=None
```

Comment:
Do we need the `default=None` here? Isn't the default value rendered based on the Python code?

I know we give the defaults in the docstring, do it through the library, but if we could leave it out to avoid duplication, we could start now.